source/js/buyers-guide/components/product-quiz/product-quiz.jsx

```diff
@@ -1,19 +1,26 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { POINTS, PRODUCTS, RESULTS } from "./data";
+import JoinUs from "../../../components/join/join.jsx";
 
 class ProductQuiz extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {
+    this.state = this.getInitialState();
+    this.smallTextClass = "text-font-sans tw-text-xs";
+    // Three steps in total: Select products -> Newsletter signup -> See results
+    this.totalSteps = 3;
+  }
+
+  getInitialState() {
+    return {
       selectedChoices: [],
       numBad: 0,
       score: 0,
-      showResults: false,
+      currentStep: 1,
+      showThankYou: false,
```

Comment:
@danielfmiranda if you wanna make the thank you screen show just for QAing purpose, you can set `currentStep` to 2 and `showThankYou` to false